openmicroscopy/bio-formats-examples
# Merge bio-formats-examples and existing history

This is https://github.com/rleigh-codelibre/bio-formats-examples/tree/master with the existing `bioformats-examples` merged into it.  This links the two unrelated histories.

Done with:

```
% git commit-tree -p HEAD -p old/master -m "Merge bioformats-examples history" 1f1267405a617c3bc6120ebedcb1403afc8562b1
% git update-ref HEAD a9a02767931578dbdd3a2ea1a23a47542b2aa875
```

to manually construct a merge.  Note: this is what we did to tie ome-xml into bioformats when we merged these repositories.

See also: https://github.com/rleigh-codelibre/bio-formats-examples/compare/merge-history?expand=1 which is the same branch in the other repository, so you can see the inverse set of commits from the other history (which shows how little of interest there is in bioformats-examples), and also the CI builds:

- [travis](https://travis-ci.org/rleigh-codelibre/bio-formats-examples/builds/349528760?utm_source=github_status&utm_medium=notification)
- [appveyor](https://ci.appveyor.com/project/rleigh-codelibre/bio-formats-examples/build/1.0.8)

Rather than merging this pull request, it might be better to push the branch directly.  Because the PR here is itself the merge commit, there's little point in doing a second merge.

Testing:

- Check the added maven and gradle build logic.
- Check all builds are green above.
- Check the git history graph with gitk / git log --graph
